### PR TITLE
Add POJO&DAO for Hosts and Categories, and some changes to project

### PR DESCRIPTION
**Pull request by NekodRider**

### CODE REVIEW

File: AirEats/java/aireats/model/Hosts.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package aireats.model;
+
+public class Hosts {
+    protected Integer HostId;
+    protected String HostName;
+
+    public Hosts(Integer hostId, String hostName) {
+        super();
```
Comment (rainbowhuanguw) [inline AirEats/java/aireats/model/Hosts.java]: this line is unnecessary

File: AirEats/java/aireats/dal/CategoriesDao.java [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+    public Categories create(Categories category) throws SQLException {
+        String insertCategory = "INSERT INTO Categories(RestaurantId,Categories) VALUES(?,?)";
+        Connection conn = null;
+        PreparedStatement insertStmt = null;
+        try {
+            conn = connectionManager.getConnection();
+            insertStmt = conn.prepareStatement(insertCategory);
+            insertStmt.setString(1, category.getRestaurantId());
+            StringBuilder categoryListStr = new StringBuilder();
```
Comment (rainbowhuanguw) [inline AirEats/java/aireats/dal/CategoriesDao.java]: replace line 38 to 43 with 
```String categoryStr = String.join(", ", category.getCategoryList());```

File: AirEats/java/aireats/dal/CategoriesDao.java [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+    }
+
+    public Categories updateCategories(Categories category, List<String> newCategories) throws SQLException {
+        String updateCategories = "UPDATE Categories SET Categories=? WHERE RestaurantId=?;";
+        Connection connection = null;
+        PreparedStatement updateStmt = null;
+        try {
+            connection = connectionManager.getConnection();
+            updateStmt = connection.prepareStatement(updateCategories);
+            StringBuilder categoryListStr = new StringBuilder();
```
Comment (rainbowhuanguw) [inline AirEats/java/aireats/dal/CategoriesDao.java]: same here, replace with string join statement

File: AirEats/java/aireats/model/Categories.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
+package aireats.model;
+
+import java.util.List;
+
+public class Categories {
+    protected String RestaurantId;
+    protected List<String> CategoryList;
+
+    public Categories(String restaurantId, List<String> categoryList) {
+        super();
```
Comment (rainbowhuanguw) [inline AirEats/java/aireats/model/Categories.java]: this line is unnecessary

**Review by rainbowhuanguw — CHANGES_REQUESTED** [review]

**Comment by NekodRider** [comment]
@rainbowhuanguw 
- removed auto-generated `Super()`
- use `String.join()` instead of `StringBuilder` in `Categories`
- also fix wrong path and empty dir

File: AirEats/src/main/java/aireats/model/Categories.java [hunk 5]
```diff
@@ -0,0 +1,44 @@
+package aireats.model;
```
Comment (rainbowhuanguw) [inline AirEats/src/main/java/aireats/model/Categories.java:1]: change class names Categories, Hosts, YelpUsers to singular form

**Review by rainbowhuanguw — APPROVED** [review]

**Pull request merged**